projects/components/src/utils/test/widget-object/cypress-widget-finder.ts

```diff
@@ -35,23 +42,6 @@ export class CypressWidgetObjectFinder {
         const search = ancestor ? cy.get(ancestor).find(tagName) : cy.get(tagName);
         const root = search.as(id);
         const widget = new widgetConstructor(new CypressLocatorDriver(root, true, id));
-        return (widget as any) as CorrectReturnTypes<InstanceType<C>, Chainable>;
+        return (widget as any) as CorrectReturnTypes<InstanceType<C>, T>;
     }
 }
-
-/**
```

Comment:
Instead of just deleting the utility, let's move it to somewhere that only unit tests can access and vcd_ui can just copy paste the approach